core/providers/vllm/chat_test.go

```
@@ -108,3 +108,184 @@ func keys(m map[string]interface{}) []string {
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		body, err := io.ReadAll(r.Body)
+		if err != nil {
+			http.Error(w, "read error", http.StatusInternalServerError)
+			return
+		}
+		if err := json.Unmarshal(body, &capturedBody); err != nil {
+			http.Error(w, "json error", http.StatusInternalServerError)
+			return
+		}
+		w.Header().Set("Content-Type", "application/json")
+		fmt.Fprint(w, `{
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprint` is not checked

(errcheck)